Automattic/wp-calypso
# A4A: Show placeholder when team member list is fetching.

The primary objective of this pull request is to display a placeholder while we are retrieving the member list. As it has become a common pattern to show a page placeholder, I have decided to create a reusable one that can be used as a generic loader.


https://github.com/user-attachments/assets/8520da4e-4d96-471e-8ac4-c098ef5adeab



Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1022

## Proposed Changes

* The first step was to create a reusable page placeholder. We already have a good placeholder on the landing page, and using the same design for the rest of the page makes sense. This PR extracts those presentations into a reusable component (`PagePlaceholder`) so we can reuse it on other pages.
* We've also updated the landing page to now use the reusable page placeholder along with the team section pages.

## Why are these changes being made?

* There is a tendency for the team member list page to flicker due to premature rendering, which makes the UI experience unpolished. For that, we can show a placeholder while we fetch the list.

## Testing Instructions

* Use the A4A live link and go to the `/team` page.
* Confirm you can see the placeholder when the page does the initial fetch.

## Pre-merge Checklist
